desktop/src/main/java/bisq/desktop/components/controls/WaitingState.java

```diff
@@ -0,0 +1,10 @@
+package bisq.desktop.components.controls;
```

Comment:
I think that should be moved inside into the bisq_easy components package as its not generic but special for bisq easy.

Reply:
Good point! Should we make the `WaitingAnimation` specific to Bisq Easy then as well? since it depends on this state to decide the `innerIcon`.
Another option could be that we keep just a record of all the possible `WaitingStates` for any protocol in common even if we don´t necessarily use them all.

Reply:
> Good point! Should we make the WaitingAnimation specific to Bisq Easy then as well? since it depends on this state to decide the innerIcon.

Yes, I guess thats better.

> Another option could be that we keep just a record of all the possible WaitingStates for any protocol in common even if we don´t necessarily use them all.

I fear thats too premature as long we dont have the other protocols. Over time we will surely generalize things...